docs/osf-trivia-list/deployment.md

```diff
@@ -154,7 +154,7 @@ Note: flashrom can never write if the flash chip isn't found automatically.
 
 that means your flashrom version is incorrect. Follow the procedure for
 building the right flashrom is described in `Build flashrom` section in the
-[Initial deployment documentation](initial-deployment.md#initial-deployment-manually).
+[Initial deployment documentation](#how-to-install-dasharo-flashrom-fork).
```

Comment:
This link does not exist

Reply:
@miczyg1 true, it depends on https://github.com/Dasharo/docs/pull/513

Reply:
Maybe worth cascading the PRs? or separate it